From a9ed8e1b467a886104094cb31a275a6fac93a136 Mon Sep 17 00:00:00 2001
From: Limeng <Meng.Li@windriver.com>
Date: Sat, 29 Sep 2018 15:44:39 +0800
Subject: [PATCH] arm64: dts: armada-8040-db: disable dts node sd_mmc_reg0

There is no SD slot on armada-8040-db board, so disable the
sd card plug detection dts node.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-db.dts |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db.dts b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
index a37bae8..4a51b66 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db.dts
@@ -493,3 +493,7 @@
 &cps_pcie1 {
 	status = "okay";
 };
+
+&sd_mmc_reg0 {
+	status = "disabled";
+};
-- 
1.7.9.5

